apps/discord-bot/src/services/mojang-api.service.ts

```diff
@@ -0,0 +1,72 @@
+import axios, { AxiosInstance } from 'axios';
+import { Service } from 'typedi';
+import { ErrorMessage } from '../error.message';
+
+interface AshconResponse {
+  uuid: string;
+  username: string;
+  username_history: { username: string; changed_at?: string }[];
+  textures: {
+    custom: boolean;
+    slim: boolean;
+    skin: { url: string; data: string };
+    cape?: { url: string; data: string };
+    raw: { url: string; data: string };
+  };
+}
+
+interface AshconErrorResponse {
+  code: number;
+  error: string;
+  reason: string;
+}
+
+@Service()
+export class MojangApiService {
+  private ashcon: AxiosInstance;
+
+  public constructor() {
+    this.ashcon = axios.create({
+      baseURL: 'https://api.ashcon.app/mojang/v2/user/',
+      timeout: 10000,
+    });
+  }
+
+  public async getName(uuid: string) {
```

Comment:
```suggestion
  public async getPlayer(uuid: string) {
```

Make this return all the mojang player data, not just the name